pillarbox-analytics/docs/README.md

```diff
@@ -1,95 +1,122 @@
     sourceKey = AnalyticsConfig.SOURCE_KEY_SRG_DEBUG,
-    userConsent = initialUserConsent
+    userConsent = userConsent,
 )
 
-initSRGAnalytics(config = config)
+initSRGAnalytics(config)
 ```
 
 Or it can be updated at any time using the following code snippet:
+
 ```kotlin
-val updatedUserConsent = UserConsent(
-    comScore = ComScoreUserConsent.DECLINED, // or ComScoreUserConsent.ACCEPTED
-    commandersActConsentServices = listOf("service1_id", "service2_id")
+val userConsent = UserConsent(
+    comScore = ComScoreUserConsent.DECLINED,
+    commandersActConsentServices = listOf("service1_id", "service2_id"),
 )
 
-SRGAnalytics.setUserConsent(updatedUserConsent)
+SRGAnalytics.setUserConsent(userConsent)
 ```
 
-User consent values will be updated with the next analytics event.
+The updated values will be sent with the next analytics event.
 
 ### Send page view
 
-To send a page view use `SRGAnalytics.sendPageView`. It will trigger a CommandersAct and a Comscore page view event directly.
+To send a page view, use [SRGAnalytics.sendPageView][ch.srgssr.pillarbox.analytics.SRGAnalytics.sendPageView]. It will send the event to both
+Commanders Act and ComScore.
 
 ```kotlin
-val commandersActEvent = CommandersActPageView(name = "main", type = "tbd", levels = listOf("app", "pillarbox"))
-val comScoreEvent = ComScorePageView(name = "main")
-SRGAnalytics.sendPageView(commandersAct = commandersActEvent, comScore = comScoreEvent)
-```
+val commandersActPageView = CommandersActPageView(
+    name = "page_name",
+    type = "page_type",
+    levels = listOf("level1", "level2"),
+)
 
-In the case of a multi pane view each pane view can send a page view. It is useful then reusing view from single pane view inside the multi pane view.
+val comScorePageView = ComScorePageView(name = "page_name")
 
-For Android Auto application it is not recommended to send page view.
+SRGAnalytics.sendPageView(
+    commandersAct = commandersActPageView,
+    comScore = comScorePageView,
+)
+```
+
+In the case of a multi-pane view, each pane view can send a page view. It is useful when reusing views from a single pane view inside the multi-pane
+view.
+For Android Auto applications, it is not recommended to send page view.
 
 ### Send event
 
-Events are application events the analytics team of the application want to track. It could be click event, user choice etc..
+Events are application events that the analytics team wants to track. It could be a click event, a user choice, etc...
 
 ```kotlin
-SRGAnalytics.sendEvent(CommandersActEvent(name = "event"))
+val commandersActEvent = CommandersActEvent(name = "event")
+
+SRGAnalytics.sendEvent(commandersActEvent)
 ```
+
+# Package ch.srgssr.pillarbox.analytics
+
+Top-level entry point for managing analytics in Pillarbox for SRG SSR applications.
+
+# Package ch.srgssr.pillarbox.analytics.commandersact
+
+Commanders Act specific classes.
+
+# Package ch.srgssr.pillarbox.analytics.comscore
+
+ComScore specific classes.
```

Comment:
I assume it is dokka specific ?

Reply:
Indeed. The format is documented here: https://kotlinlang.org/docs/dokka-module-and-package-docs.html

A simple example:
```markdown
# Module my-module

The documentation for the `my-module` module.

# Package com.example.my.package

The documentation for the `com.example.my.package` package.
```